src/components/flow-item/flow-item.tsx

```diff
@@ -302,7 +301,6 @@ export class FlowItem implements InteractiveComponent, LoadableComponent {
           heading={heading}
           headingLevel={headingLevel}
           heightScale={heightScale}
-          intlBack={intlBack}
```

Comment:
Looks like `intlBack` was removed from `calcite-flow-item` in this PR as well. Copy/paste and Search/replace gets the best of us :sweat_smile:

Reply:
The flow-item uses the panel internally. Since this prop has been removed in this PR I need to no longer set it on the panel.

Reply:
Ahh I missed `intlBack` being used in `getBackLabel`, I thought it wasn't being used at all anymore. Thanks for clarifying